diags_rean.csh

```diff
@@ -22,6 +22,8 @@
 #PBS  -o diags_rean.eo
 #PBS  -j oe 
 #PBS  -k eod 
+# #PBS  -W depend=afterok:2932435.chadmin1.ib0.cheyenne.ucar.edu
```

Comment:
Is this something we want to check-in?

Reply:
I just saw this is commented out. It's up to you if you'd like to keep it, please just verify either way.